pybit/_websocket_stream.py

```diff
@@ -205,44 +255,61 @@ def _auth(self):
         )
 
         # Authenticate with API.
-        self.ws.send(
+        self._send(
             json.dumps(
                 {"op": "auth", "args": [self.api_key, expires, signature]}
             )
         )
 
-    def _on_error(self, error):
+    def _on_error(self, _, error):
         """
         Exit on errors and raise exception, or attempt reconnect.
         """
-        if type(error).__name__ not in [
-            "WebSocketConnectionClosedException",
-            "ConnectionResetError",
```

Comment:
"ConnectionResetError" cannot be thrown from the `websocket` package. Perhaps, this exception is from `websockets` package. I assume it was used back in the days, but then switched to `websocket`, but this exception remained here. This is one reason why it's not a very good practice to check the type this way.